internal/encryption/keyprovider/gcp_kms/provider.go

```diff
@@ -67,13 +68,15 @@ func (p keyProvider) Provide(rawMeta keyprovider.KeyMeta) (keyprovider.Output, k
 			Cause:   err,
 		}
 	}
+	log.Printf("[DEBUG] GCP KMS: encrypted state : %s", encryptedKeyData.Name)
```

Comment:
```suggestion
	log.Printf("[DEBUG] GCP KMS: encrypted state: %s", encryptedKeyData.Name)
```

Just to be consistent with the rest of log messages.